src/doc/tarpl/lifetime-elision.md

```diff
@@ -0,0 +1,64 @@
+% Lifetime Elision
+
+In order to make common patterns more ergonomic, Rust allows lifetimes to be
+*elided* in function signatures.
+
+A *lifetime position* is anywhere you can write a lifetime in a type:
+
+```rust,ignore
+&'a T
+&'a mut T
+T<'a>
+```
```

Comment:
N.B. There is also `Box<Trait+'a>`, which is a lifetime in a type, but has a different role (that of a bound) than the role of `'a` in `T<'a>`.

But then again, this might not be considered an elision position.

See in particular:
   https://github.com/rust-lang/rfcs/pull/1177#issuecomment-124283746